plugins/source/aws/resources/services/lambda/functions_fetch.go

```diff
@@ -35,13 +35,6 @@ func fetchLambdaFunctions(ctx context.Context, meta schema.ClientMeta, parent *s
 			})
 
 			if err != nil {
-				if c.IsNotFoundError(err) || c.IsAccessDeniedError(err) {
-					c.Logger().Warn().Err(err).Msg("Failed to get function")
-					res <- &lambda.GetFunctionOutput{
-						Configuration: &f,
-					}
-					continue
-				}
```

Comment:
Without this, a single permission error or function not found will stop all fetching for the entire region

Reply:
> Left some comments. Overall we really need to consider if we should stop fetching on 403/404/NotFound errors... Because it will make it much harder for users to grab data as in order to grab data about any resource you have to 0 errors for the entire resource

It should be the default behaviour in the SDK otherwise will be implementing error parsing all the time and running after more 404 and so on. So basically all resolvers should be called no matter what and for list/detail we should use the preResourceResolver

Reply:
So should that change be a blocker for v2 release? If we leave this pr as it is many users won't be able to grab resources